Bottom: 3820026fb7a4b78db9f55ffd1aa64af86b514c95
Top:    1f28699d1b7caf40248908169277f169f3a13865
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:16:07 +0000

Refresh of bentley-ottann-test

---

diff --git a/src/borast/borast-bentley-ottmann.c b/src/borast/borast-bentley-ottmann.c
index 7a0d1ac..f9b42d4 100644
--- a/src/borast/borast-bentley-ottmann.c
+++ b/src/borast/borast-bentley-ottmann.c
@@ -48,10 +48,16 @@
 #include <glib.h>
 
 #include "polygon.h"
-#include <GL/gl.h>
 #include "hid_draw.h"
+
+#ifdef WIN32
+#   define WIN32_LEAN_AND_MEAN 1
+#endif
+
+#include <GL/gl.h>
 #include "hid/common/hidgl.h"
 
+
 #define _borast_error(x) (x)
 
 #define DEBUG_PRINT_STATE 0
